testutils/resourcebuilder/machine/v1beta1/gcp_provider_spec.go

```diff
@@ -65,15 +75,7 @@ func (m GCPProviderSpecBuilder) Build() *machinev1beta1.GCPMachineProviderSpec {
 		CanIPForward: false,
 		ProjectID:    "openshift-cpms-unit-tests",
 		Region:       "us-central1",
-		Disks: []*machinev1beta1.GCPDisk{
-			{
-				AutoDelete: true,
-				Boot:       true,
-				Image:      "projects/rhcos-cloud/global/images/rhcos-411-85-202205101201-0-gcp-x86-64",
-				SizeGB:     128,
-				Type:       "pd-ssd",
-			},
-		},
+		Disks:        m.disks,
```

Comment:
This too

Reply:
Hmmmm... this is decidedly not alphabetical order  (M..->U..->T..->D..). Let's leave the disks entry where it was previously??